src/notifications.js

```diff
@@ -43,24 +43,13 @@ export default class NotificationsManager {
         return;
       }
 
-      if (responseData && typeof responseData === 'string') {
-        switch (responseData.trim()) {
-          case "ErrPathHasUnsupportedExtension":
-            vscode.window.showWarningMessage(`Code Finder does not support the \`${path.extname(filename)}\` file extension yet.`);
-            return;
-          case "ErrPathNotInSupportedProject":
-            vscode.window.showWarningMessage(`The file ${filename} is not in any Git project. Code finder only works inside Git projects.`);
-            return;
-          case "ErrProjectStillIndexing":
-            vscode.window.showWarningMessage(
-              "Kite is not done indexing your project yet. Please wait for the status icon to switch to ready before using Code Finder."
-            );
-            return;
-          case "ErrEmptyLine":
```

Comment:
`ErrEmptyLine`is actually thrown in-editor instead of returned from kited (case for most editors). Are we planning to have the api check for empty lines?

Reply:
(The other option would be to make the change https://github.com/kiteco/vscode-plugin/blob/3f6bdf8c33b364be7586eeef46fd6a7b2a7fc153/src/kite.js#L236 to use this new message api, with all the caveats of doing it that way...)